src/node.js

```diff
@@ -335,23 +335,27 @@
         scheduleMicrotasks();
     }
 
+    function _tryCallback(callback) {
+      var threw = true;
+      try {
+        callback();
+        threw = false;
+      } finally {
+        if (threw)
+          tickDone();
```

Comment:
I suppose `finally` not requires stack-trace collecting, but `catch` does. But it's just my thoughts.